src/core/polymer.hpp

```diff
@@ -32,107 +32,43 @@
 
 #include "PartCfg.hpp"
 #include "particle_data.hpp"
+#include "utils/Vector.hpp"
 
-/*************************************************************
- * Functions                                                 *
- * ---------                                                 *
- *************************************************************/
+Utils::Vector3d random_position(const std::function<double()> &generate_rn);
+Utils::Vector3d random_unit_vector(const std::function<double()> &generate_rn);
 
-/** C implementation of 'mindist \<part_id\> \<r_catch\>',<br>
- *  which returns the size of an array \<ids\> of indices of particles which are
- *  less than \<r_catch\> away from the position of the particle \<part_id\>.
+/** Returns the miminum distance between position pos and all existing
+ * particles.
  */
-int mindist3(PartCfg &, int part_id, double r_catch, int *ids);
+double mindist(PartCfg &partCfg, Utils::Vector3d pos);
```

Comment:
```Utils::Vector3d const &pos```

Reply:
Makes clang:6.0 [CI test fail](https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/-/jobs/104488):
```
/builds/espressomd/espresso/src/core/polymer.hpp:41:34: error: parameter 'pos' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions [readability-avoid-const-params-in-decls,-warnings-as-errors]
double mindist(PartCfg &partCfg, const Utils::Vector3d pos);
                                 ^~~~~~~~~~~
```

Reply:
the ampersand was missing, if you add it the error will go away

Reply:
Added it just now. Thanks @jngrad!